src/main/java/salesforce/ui/pages/tables/TableGroup.java

```diff
@@ -0,0 +1,154 @@
+     */
+    public void loadTables() {
+        waitForPageLoaded();
+        webDriverManager.setTableWaitMode();
+        boolean thereAreResults = thereAreResults();
+        webDriverManager.setDefaultWaitMode();
+        if (thereAreResults) {
+            int totalTables = getTablesWithHidenOnes();
+            for (int i = 1; i <= totalTables; i++) {
+                webDriverManager.setTableWaitMode();
+                boolean validElement = validElement(i);
+                webDriverManager.setDefaultWaitMode();
+                if (validElement) {
+                    String baseLocator = getValidTableContainerSelector.replaceAll("%d", String.valueOf(i));
+                    String featureName = webElementAction.getElement(String.format(getValidTableContainerSelector, i)
+                            .concat(titleNameLocator)).getText();
+                    tables.add(new Table(webDriverManager, baseLocator, featureName));
+                }
+            }
+        }
+    }
+
+    /**
+     * Returns all records from all tables.
+     */
+    public List<Record> getEachTableRecords() {
+        List<Record> records = new ArrayList<>();
+        if (!tables.isEmpty()) {
+            tables.stream().forEach(table -> records.addAll(table.getRecords()));
+        }
+        return records;
+    }
+
+    private int getTablesWithHidenOnes() {
+        return webElementAction.getElements(getAllTableContainersSelector).size();
+    }
+
+    private boolean validElement(final int i) {
+        try {
+            WebElement webElement = webElementAction.getElement(String.format(getValidTableContainerSelector, i));
+            if (webElement != null) {
+                return true;
+            } else {
+                return false;
+            }
+        } catch (Exception e) {
+            return false;
+        }
+    }
+
+    /**
+     * Returns a Table with title matching the name.
+     *
+     * @param name represents the  table title name
+     * @return a Table
+     */
+    public Table getTableFromName(final String name) {
+        return tables.stream().filter(c -> name.equals(c.getName())).findFirst().get();
+    }
+
+    /**
+     * Returns a list with all the values for the specified column in the tables.
+     *
+     * @param columnName represents a column from the tables
+     * @return a List
+     */
+    public List<String> getTableValuesByColumnName(final String columnName) {
+        List<String> columnValues = new ArrayList<>();
+        tables.stream().forEach(table -> columnValues.addAll(table.getColumnValuesByColumnName(columnName)));
+        return columnValues;
+    }
+
+    /**
+     * Checks if there are table results.
+     *
+     * @return a boolean
+     */
+    public boolean thereAreResults() {
+        try {
+            webElementAction.getElements(getAllValidTableContainersSelector);
+            return true;
+        } catch (Exception e) {
+            e.printStackTrace();

```

Comment:
It would throw a message when it catch a exception

Reply:
Added that.